modules/nw-sriov-concept-dpdk-line-rate.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-sriov-example-dpdk-line-rate_{context}"]
+= Overview of achieving a specific DPDK line rate
+
+To achieve a specific Data Plane Development Kit (DPDK) line rate, deploy a Node Tuning Operator, configure Single Root I/O Virtualization (SR-IOV), and tune DPDK settings for:
+
+- isolated CPUs
+- hugepages
+- topology scheduler
+
+[NOTE]
+====
+In previous versions of {product-title}, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In {product-title}{product-version} 4.11, this functionality is part of the Node Tuning Operator.
```

Comment:
You could replace "OpenShift applications" with "`{product-title}` applications".

Reply:
Done.